app/uk/gov/hmrc/emcstfe/models/request/SubmitCancellationOfMovementRequest.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfe.models.request
+
+import uk.gov.hmrc.emcstfe.models.auth.UserRequest
+import uk.gov.hmrc.emcstfe.models.cancellationOfMovement.SubmitCancellationOfMovementModel
+
+import scala.xml.Elem
+
+case class SubmitCancellationOfMovementRequest(body: SubmitCancellationOfMovementModel)
+                                              (implicit request: UserRequest[_]) extends ChrisRequest {
+  override def exciseRegistrationNumber: String = request.ern
+
+  private val arcCountryCode: String = body.exciseMovement.arc.substring(2, 4)
+
+  val messageRecipient: String = NDEA ++ arcCountryCode
+  val messageSender: String = NDEA ++ GB
```

Comment:
```
  val messageSender: String = NDEA ++ GB
```
I'm not sure if this is correct but I couldn't find somewhere to pull it from like other models have done

Reply:
I think there's something on Matt's branch for this so it's probably worth leaving it for now and updating it with his change

Reply:
@r-melvin you'll need to copy what's currently done for legacy here, I'll send you a link to the code in Slack as it's on a Private repo.